### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.4.3 Contrast (Minimum)

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#652]
(Mary Jo) 

- Same as 1.4.2, Audio control is about the content - not the view. Any "content" that does not meet this success criterion can interfere with a user's ability to use the whole software application. Don't think it is necessary to take this to a more granular "view" concept. 
- Phil's issue https://github.com/w3c/wcag2ict/issues/638 should solve this problem.

(Chris L)

-  Reading through https://github.com/w3c/matf/issues/28, the discussion appears to be more related to thresholds for failing per criteria set vs. whether or not the SC applies to mobile. The "work in progress" should make sure that they are separating out these two goals to align with the MATF charter deliverables and possibly explore techniques / failures as a secondary outcome of their analysis.
